internal/build/build.go

```diff
@@ -25,11 +25,8 @@ var (
 	// RepositoryName is the remote GitHub repository for the releases
 	RepositoryName = "app-services-cli"
 
-	// TermsReviewEventCode is the event code used when checking the terms review
-	TermsReviewEventCode = "register"
-
-	// TermsReviewSiteCode is the site code used when checking the terms review
-	TermsReviewSiteCode = "ocm"
+	// TermsReviewSpecURL Url used to download terms and conditions specification
+	TermsReviewSpecURL = "https://raw.githubusercontent.com/redhat-developer/app-services-ui/main/static/configs/terms-conditions-spec.json"
```

Comment:
Considering this is hotfix I think we could merge it with that location and switch over to cdn version after?

Reply:
I think we could have three levels of fallback:

1. Try to load from CDN, if it fails go to number 2:
2. Try to load from GitHub, if it fails to go number 3:
3. Load the local spec file.

If it is cached, doing 3 checks should not be a problem.

Reply:
I think we could have three levels of fallback:

1. Try to load from CDN, if it fails go to number 2:
2. Try to load from GitHub, if it fails to go number 3:
3. Load the local spec file.

If it is cached, doing 3 checks should not be a problem.

Reply:
Having two sources (one of which we do not control - github) is going to be tricky from compliance/firewal rules point of view. We use Akamai as CDN for console so this should be very reliable way. Having second URL would be problematic from debugging point of view (in case issues happens). We have fallback in code right now